### ISSUE TITLE
aria-ng-gui@2.0.1: hash check failed

### ISSUE THREAD

**Issue by Volipy** [lukesampson/scoop-extras#2912]
Checking hash of AriaNgGUI-win32-x64.zip ... ERROR Hash check failed!
App:         extras/aria-ng-gui
URL:         https://github.com/Xmader/aria-ng-gui/releases/download/v2.0.1/AriaNgGUI-win32-x64.zip
First bytes: 50 4B 03 04 14 00 08 00
Expected:    1080242f41a89b54033eb7122d7385c30ab18430371876ba1bb37463942131c8
Actual:      a7e8b9a4a38c8e0779861becf58aa948a5ff45b1e782932007e2ff0074577ac3

**Comment by Ash258** [comment]
Supressed by eaa70c55c42df0d9f19bf76909811621ee0c5b92
